applications/ShallowWaterApplication/custom_processes/write_from_sw_at_interface_process.cpp

```diff
@@ -119,8 +96,8 @@ void WriteFromSwAtInterfaceProcess<TDim>::ReadAndSetValues(
     double vertical_velocity = 0.0;
     double topography = 0.0;
     
-    velocity = rNode.FastGetSolutionStepValue(VELOCITY);
-    momentum = rNode.FastGetSolutionStepValue(MOMENTUM);
+    noalias(velocity) = rNode.FastGetSolutionStepValue(VELOCITY);
```

Comment:
just in case, using `noalias` is notably faster than not using it. There is a key aspect, when using noalias one must ensure that the size of the vector is correct before assignation (here velocity is an array_1d with fixed size --> ok)